packages/base/src/api/client.ts

```diff
@@ -64,6 +70,17 @@ export abstract class BaseApiClient {
         const api = await this.refresh();
         return await fn(api);
       }
+      // Cloudflare error
+      if (error.response && error.response.status === 520 && error.response.data.includes('Cloudflare')) {
+        this.api = undefined;
+
+        const headersOverride: Record<string, string> = {
+          'Connection': 'upgrade',
+          'Upgrade': 'HTTP/2.0'
+        };
+        const api = await this.refresh({ headers: headersOverride });
```

Comment:
Do you think if we should retry three times with exp backoff logic?

Reply:
I like the idea, but we would need to add [axios-retry](https://www.npmjs.com/package/axios-retry) for a clean implementation

Reply:
> I like the idea, but we would need to add [axios-retry](https://www.npmjs.com/package/axios-retry) for a clean implementation

💯

Reply:
We can iterate it in the next version, for now we can push it through.